react/PageBlock/PageBlock.js

```diff
@@ -0,0 +1,13 @@
+import styles from './PageBlock.less';
+import React from 'react';
+import classnames from 'classnames';
+
+export default function PageBlock({ children, className }) {
+  return (
+    <div className={classnames(className)}>
```

Comment:
Did we see this return "false", should this be `classnames({[className]: className})`?